crates/sui-config/src/genesis.rs

```diff
@@ -136,6 +137,19 @@ impl Genesis {
         result
     }
 
+    pub fn clock(&self) -> Clock {
+        let clock = self
+            .objects()
+            .iter()
+            .find(|o| o.id() == sui_types::SUI_CLOCK_OBJECT_ID)
```

Comment:
Does it matter that this is potentially inefficient?

Reply:
Not so much -- this is just used in tests.